addon/components/search-street.js

```diff
@@ -12,6 +12,22 @@ export default class SearchStreetComponent extends ValidatedInput {
 
   @tracked searchTerm = "";
 
+  get errors() {
+    const errors =
+      (this.args.model &&
+        get(
+          this.args.model,
+          `error.${this.args.options.validationPath}.validation`,
+        )) ??
+      [];
```

Comment:
IMO this is almost unreadable, would prefer this or ternary.
```suggestion
    if (!this.args.model) {
      return []
    }
    
    const errors = get(
      this.args.model,
      `error.${this.args.options.validationPath}.validation`,
    );
```

Reply:
Also is it possible that we now ignore errors because we now look only at the `validationPath`? Should we maybe merge them ? `return [...errors, ...super.errors]`?

Reply:
There won't be any, since there are not validations for it.